src/components/Hero/Hero.stories.mdx

```diff
@@ -156,76 +179,149 @@ through `Heading`'s `level="one"`.
 
 ## All Variations
 
+Each `Hero` variation can be rendered through the `heroType` prop.
+
+### Primary
+
+The `"primary"` hero type should be used with the `backgroundImageSrc` and
```

Comment:
There are a number of optional props that were not mentioned through these descriptions. For example, off the top of my head, I know `tertiary` can accept a background color. Were those purposely omitted?

Reply:
Nope. I just don't know all of them!

Reply:
I notice there might be an issue... we don't use the `locationsDetails` prop anywhere or have examples. When testing this, it doesn't render right. This could be another ticket. In this example it's hard to see but it's on the right side (a warning also says it's only allowed in the "primary" hero type):

<img width="717" alt="Screen Shot 2022-05-06 at 11 01 10 AM" src="https://user-images.githubusercontent.com/1280564/167159711-1968056a-7fed-44ae-a401-1ab80feb83b5.png">

Reply:
I agree. This does not need to be completed before v1.